Joint_1: {p: 10.0, i: 1, d: 1}
Joint_2: {p: 10, i: 1, d: 1}
Joint_3: {p: 10, i: 1, d: 1}
